tests/integration/golang/helpers/s3.go

```diff
@@ -31,3 +35,51 @@ func NewS3Client(endpoint string) (*s3.Client, error) {
 		o.UsePathStyle = true
 	}), nil
 }
+
+func CreateBuckets(s3Client *s3.Client) error {
+	for _, bucket := range testBuckets {
+		_, err := s3Client.CreateBucket(context.Background(), &s3.CreateBucketInput{
+			Bucket:                    aws.String(bucket),
+			CreateBucketConfiguration: &types.CreateBucketConfiguration{},
+		})
+		if err != nil {
+			return fmt.Errorf("failed to create bucket '%s': %v", bucket, err)
```

Comment:
please either use `%w` to wrap the error, or use `eris.Wrapf`

same remark further in this file

Reply:
actually, please use `eris` for consistency

Reply:
will do